sources/engine/Xenko.Graphics/DescriptorSet.cs

```diff
@@ -66,17 +66,22 @@ public void SetSamplerState(int slot, SamplerState samplerState)
         /// <param name="size">The constant buffer view size.</param>
         public void SetConstantBuffer(int slot, Buffer buffer, int offset, int size)
         {
-            HeapObjects[DescriptorStartOffset + slot] = new DescriptorSetEntry(buffer, offset, size);
+            HeapObjects[DescriptorStartOffset + slot] = new DescriptorSetEntry(buffer);
```

Comment:
Please revert this, might be used later (D3D11.2 or OpenGL supports cbuffer offsets/size as well)

Reply:
yes, just noticed myself that i forgot to revert it.